api/src/api/application_alpha/application_route.py

```diff
@@ -90,13 +91,33 @@ def application_form_get(
     )
     logger.info("GET /alpha/applications/:application_id/application_form/:app_form_id")
 
+    validation_warnings = []
```

Comment:
Can we put the validation warnings in the `get_application_form` function like we have for the PUT endpoint?